File: src/main/java/no/nav/syfo/repository/InntektsmeldingMeta.java
package no.nav.syfo.repository;

import lombok.Builder;
import lombok.Getter;
import lombok.Value;
import no.nav.syfo.domain.Periode;

import java.time.LocalDateTime;
import java.util.List;

@Value
@Builder
@Getter
public class InntektsmeldingMeta {
    private String uuid;
    private String aktorId;
    private String sakId;
    private String journalpostId;
    private String orgnummer;
    private String arbeidsgiverPrivat;
    private LocalDateTime behandlet;
    private List<Periode> arbeidsgiverperioder;
}

File: src/main/kotlin/no/nav/syfo/domain/InntektsmeldingMeta.kt
package no.nav.syfo.domain

import java.time.LocalDateTime

data class InntektsmeldingMeta (
    val uuid: String? = null,
    val aktorId: String,
    val sakId: String,
    val journalpostId: String,
    val orgnummer: String? = null,
    val arbeidsgiverPrivat: String? = null,
    val behandlet: LocalDateTime? = null,
    val arbeidsgiverperioder: List<Periode> = emptyList()
)
